tests/Integration/Behaviour/Features/Context/CurrencyFeatureContext.php

```diff
@@ -63,12 +66,43 @@ public function cleanCurrencyFixtures()
         $this->currencies = [];
     }
 
+    /**
+     * @Given currency :reference with isoCode :isoCode exists
+     */
+    public function createCurrencyWithIsoCode($reference, $isoCode)
+    {
+        //Currency::getIdByIsoCode only returns not deleted currency so we check the storage to avoid duplicate contents
```

Comment:
```suggestion
        /**
         * Currency::getIdByIsoCode only returns not deleted
         * currency so we check the storage to avoid duplicate contents
         */
```

Reply:
Done